backend/src/server.ts

```diff
@@ -43,6 +50,9 @@ import env from './lib/env'
   const qualityRoutes = new QualityReportRoutes(conn, fileRoutes)
 
   const siteContactRoutes = new SiteContactRoutes(conn)
+  const userAccountRoutes = new UserAccountRoutes(conn)
+  const canUpload: PermissionType = permissionTypeFromString('canUpload')!
+  const canUploadModel: PermissionType = permissionTypeFromString('canUploadModel')!
```

Comment:
Voi käyttää suoraan arvoja, eikä muuttujia tarvitse:

```suggestion
  const canUpload = PermissionType.canUpload
  const canUploadModel = PermissionType.canUploadModel
```